### ISSUE TITLE
[1.12.2]Unknown object of FuturePack's leaves

### ISSUE THREAD

**Issue by Tronif** [mcenderdragon/Futurepack-API#473]
Version: 26.3.210

Collect leaves with silk touch tools at Tyros and become unknown objects. Texture is not prepared?
![2018-07-21_01 47 35](https://user-images.githubusercontent.com/40921102/43015049-4ad80c9e-8c89-11e8-87a1-54c2136a2fe4.png)

**Comment by mcenderdragon** [comment]
thats interresting.

**Comment by mcenderdragon** [comment]
I could not replciate your issue perse but I noticed another bug, mabe they are connected. It would be nice if you could make a screenshot in F3 while viewing the broken leaves.

**Comment by mcenderdragon** [comment]
You can try if it workis in this version. [26.3.289](https://redsnake-games.de/Downloads/fp/fp.php?t=kjUOa.g84zvbRhsgLuMslA--)

**Comment by Tronif** [comment]
OK, I got screenshot here. and I posted also FP-Latest.log.
![2018-08-06_17 56 21](https://user-images.githubusercontent.com/40921102/43707290-b335bfa4-99a2-11e8-86d4-5e9302ac28a7.png)
[FP-Latest.log](https://github.com/mcenderdragon/Futurepack-API/files/2261857/FP-Latest.log)

**Comment by Tronif** [comment]
In this version, it seems that the amount of energy of the battery pack is repeatedly increasing and decreasing even though it is not using Future pack's machines.

**Comment by mcenderdragon** [comment]
In the log you posted everyhting seems normal. The in the screenshot you look at the dropeed item but I need the placed block with the F3 menue, sorry I was not exact enough. 
Is the batterie pack connected to anything ?

**Comment by Tronif** [comment]
I was rude, I was short of understanding.
I will attach a screen shot again.
Placing leaves looks like an ordinary block.
![2018-08-06_19 58 28](https://user-images.githubusercontent.com/40921102/43713154-89c31872-99b3-11e8-92d7-68b7ddb29e82.png)


Battery packs are connected to researchers, Neon Furnace, Industrial Neon Furnace, Flash server, etc. with support cables. It seems that the phenomenon has not occurred when starting Minecraft again.

**Comment by mcenderdragon** [comment]
interesting it seems like ony the inventory item is affected. Can you press F3 + H (to activate advanced tooltips) and the make a screeenshot while hovering with the mouse over the item in inventory?

**Comment by Tronif** [comment]
I have taken a screenshot again. (F3 + H)
![2018-08-06_20 22 21](https://user-images.githubusercontent.com/40921102/43714119-ef1fdd9c-99b6-11e8-85dd-0075616ea15c.png)

**Comment by mcenderdragon** [comment]
does this also happen with vanilla tools / vanilla shears ?

**Comment by Tronif** [comment]
If using vanilla shears, Leaves is broken. It can collect with silk touch enchanted tools only.  I think Future Pack's leaves may be better not to be influenced by silk touch if possible.

**Comment by mcenderdragon** [comment]
silk touchg is completly implemnted by the mod mish is providing it, in most cases vanilla is doing it. In the version above I fixed the bug with the broken shears.

**Comment by Tronif** [comment]
OK, I will post a screenshot of collecting leaves again with vanilla shears. It seems that it operates normally here.
![2018-08-06_20 56 52](https://user-images.githubusercontent.com/40921102/43715500-a5b726a6-99bb-11e8-8866-04254af3e723.png)

**Comment by Tronif** [comment]
Leaves ID(#4862/0) seems correct at above screenshot.
Leaves ID(#4862/8 or #4862/12) are wrong?

**Comment by mcenderdragon** [comment]
I tested it and even with silk touch (tested with iron axe and shear) the leaves dropped normaly. So most likly its the issue of the tool you are using.

**Comment by Tronif** [comment]
I made silk touch tools(vanilla Iron Axe/Pickaxe/Shovel) and collected leaves again.

I noticed that there were two patterns.
If I destroy leaves block (not collected with shears) with silk touch tools, it becomes an unknown object.
next, once collected with shears, and placed it, then destroy leaves blocks with silk touch tools, it became leaves block normaly.
- silk touch tools means vanilla iron axe/pickaxe/shovels.
